src/identity/commentEncryption.test.ts

```diff
@@ -237,7 +237,10 @@ describe(checkTxsForIdentityMetadata, () => {
     const mockAccountsWrapper = {
       getWalletAddress: jest.fn((address) => address),
     }
-    await expectSaga(checkTxsForIdentityMetadata, updateTransactions(transactions))
+    await expectSaga(
+      checkTxsForIdentityMetadata,
```

Comment:
we can probably skip this saga for non Celo networks...right?

Reply:
Right 💯